Dockerfile

```diff
@@ -1,80 +1,29 @@
-FROM ubuntu:16.04 as DEV
-RUN apt-get update && apt-get install -y \
-            autoconf \
-            automake \
-            build-essential \
+FROM ubuntu:18.04
+RUN apt-get update && apt-get install -y --no-install-recommends \
```

Comment:
Since Myriad plugin for NCS is public we can include `usbutils` package that is needed for Movidius Sticks to work. 
We are already having it in binary build dockerfile to enable NCS.